crime-applications-adaptor/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/controller/CrimeApplicationController.java

```diff
@@ -13,52 +13,49 @@
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
-import uk.gov.justice.laa.crime.applications.adaptor.model.EformStagingResponse;
-import uk.gov.justice.laa.crime.applications.adaptor.model.MaatApplication;
+import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.MaatCaaContract;
+import uk.gov.justice.laa.crime.applications.adaptor.model.eform.EformStagingResponse;
 import uk.gov.justice.laa.crime.applications.adaptor.service.CrimeApplicationService;
 import uk.gov.justice.laa.crime.applications.adaptor.service.EformStagingService;
 
 @Slf4j
 @RestController
 @AllArgsConstructor
 @RequestMapping("/api/internal/v1/crimeapply")
-@Tag(name = "Crime Application Adaptor", description = "Rest API to retrieve application details from crime apply datastore")
+@Tag(name = "Crime Applications Adaptor", description = "Rest API to retrieve application details from crime apply datastore")
 public class CrimeApplicationController {
 
     private CrimeApplicationService crimeApplicationService;
-
     private EformStagingService eformStagingService;
 
     @GetMapping(value = "/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
     @Operation(description = "Retrieve application details from crime apply datastore")
     @ApiResponse(responseCode = "200",
             content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
-                    schema = @Schema(implementation = MaatApplication.class)
+                    schema = @Schema(implementation = MaatCaaContract.class)
             )
     )
-
     @ApiResponse(responseCode = "400",
             description = "Bad request.",
             content = @Content(mediaType = MediaType.APPLICATION_PROBLEM_JSON_VALUE,
                     schema = @Schema(implementation = ProblemDetail.class)
             )
     )
-
     @ApiResponse(responseCode = "500",
             description = "Server Error.",
             content = @Content(mediaType = MediaType.APPLICATION_PROBLEM_JSON_VALUE,
                     schema = @Schema(implementation = ProblemDetail.class)
             )
     )
-    public MaatApplication getCrimeApplyData(@PathVariable Long id) {
+    public MaatCaaContract getCrimeApplyData(@PathVariable long id) {
```

Comment:
is it OK to change MaatCaaContract to CrimeApplication?

Reply:
There is no reason why we can't change this, especially since the output JSON is unchanged regardless of what we name this. Will rename to CrimeApplication.

Reply:
Done, have renamed to `CrimeApplication`